conf/application.conf

```diff
@@ -24,7 +24,7 @@ play {
   }
   filters.headers {
     # Unset some of the security filters enabled in datastore.Filters
-    contentSecurityPolicy = null
+    contentSecurityPolicy = "default-src 'self'; script-src 'self' 'unsafe-eval' www.googletagmanager.com 'sha256-NA81o2FV8jh0UizSPsBz2qEGpXfap54Eghuo7vxrH8g='; style-src 'self' 'sha256-DOu86drLfwUr1Wcsx/wxfqAogK7tFvJGjVmF/300H/M=' 'sha256-47DEQpj8HBSa+/TImW+5JCeuQeRkm5NMpJWZG3hSuFU='; connect-src 'self' ws://localhost:9002 app.ovly.co; frame-src webknossos.olvy.co; img-src 'self' data:"
```

Comment:
The hashes are necessary for the inline scripts of the `main.scala.html` file. The hashes should not change afaik as the code should be static.
The variables were moved into meta tags and are loaded in the scripts by accessing the meta tags